demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb

```diff
@@ -0,0 +1,448 @@
+      "metadata": {
+        "id": "bWx5VF_LLTef"
+      },
+      "source": [
+        "## Prepare Input Data"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "yayroXhE-sos"
+      },
+      "source": [
+        "!wget https://paddlehub.bj.bcebos.com/resources/test_image.jpg"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zcrHdbJxAHh0"
+      },
+      "source": [
+        "## Create BentoService with PaddleHub Module Instantiation"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "s_T8YQRjALqg"
+      },
+      "source": [
+        "%%writefile paddlehub_service.py\n",
+        "import paddlehub as hub\n",
+        "import bentoml\n",
+        "from bentoml import env, artifacts, api, BentoService\n",
+        "import imageio\n",
+        "from bentoml.adapters import ImageInput\n",
+        "\n",
+        "\n",
+        "@env(infer_pip_packages=True)\n",
+        "class PaddleHubService(bentoml.BentoService):\n",
+        "    def __init__(self):\n",
+        "      super(PaddleHubService, self).__init__()\n",
+        "      self.module = hub.Module(name=\"deeplabv3p_xception65_humanseg\")\n",
+        "\n",
+        "    @api(input=ImageInput(), batch=True)\n",
+        "    def predict(self, images):\n",
+        "        results = self.module.segmentation(images=images, visualization=True)\n",
+        "        return [result['data'] for result in results]\n"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ESc4D_muCWNx"
+      },
+      "source": [
+        "# Import the custom BentoService defined above\n",
+        "from paddlehub_service import PaddleHubService\n",
+        "import numpy as np\n",
+        "import cv2\n",
+        "\n",
+        "# Pack it with required artifacts\n",
+        "bento_svc = PaddleHubService()"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ondQXpNCy_TV"
+      },
+      "source": [
+        "# Predict with the initialized module\n",
+        "image = cv2.imread(\"test_image.jpg\")\n",
+        "segmentation_result = bento_svc.predict([image])"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f-61QUPd6w9h"
+      },
+      "source": [
+        "### Visualizing the result"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "jNnyhPQt59ey"
+      },
+      "source": [
+        "# View the segmentation mask layer\n",
+        "\n",
+        "# If running in a Google Colab notebook, run these lines\n",
+        "from google.colab.patches import cv2_imshow\n",
+        "cv2_imshow(segmentation_result)\n",
```

Comment:
`cv2_imshow` seems to be a specific interface of google colab, it is recommended to replace it with other common interfaces.